Dockerfile

```diff
@@ -2,13 +2,15 @@ FROM python:3.6-stretch
 
 ARG ENVIRONMENT=production
 
-WORKDIR /app
+RUN useradd --create-home avraeservice
+USER avraeservice
+WORKDIR /home/avraeservice
 
-COPY requirements.txt .
-RUN pip install -r requirements.txt
+COPY --chown=avraeservice:avraeservice requirements.txt .
+RUN pip install --user --no-warn-script-location -r requirements.txt
 
-COPY . .
+COPY --chown=avraeservice:avraeservice . .
 
-COPY docker/config-${ENVIRONMENT}.py config.py
+COPY --chown=avraeservice:avraeservice docker/config-${ENVIRONMENT}.py config.py
 
-ENTRYPOINT gunicorn --workers 2 --bind 0:8000 app:app
+ENTRYPOINT .local/bin/gunicorn --workers 2 --bind 0:8000 app:app
```

Comment:
.local?

Reply:
> .local?

Fun thing with using `pip install --user`: packages are installed in `~/.local/`, so the path is `/home/avraeservice/.local/bin/gunicorn` instead of `/usr/local/bin/gunicorn`. Could also modify PATH Iguess!